impl/client/src/main/javascript/web/pentaho/visual/role/property.js

```diff
@@ -915,26 +910,13 @@ define([
               var key = isQuant ? roleAttr.keyQuantitative : roleAttr.keyQualitative;
```

Comment:
See, here, `key = roleAttr.name`. No need for `isQuant`, `keyQuantitative` and `keyQualitative`.